Fsd.Lukasz.Cs2/Fsd.Lukasz.Cs3/Models/Shop.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Fsd.Lukasz.Cs3;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fsd.Lukasz.Cs3.Models
+{
+    public class Shop : IShop
+    {
+        bool IShop.Leave { get; set; }
```

Comment:
to że IDE tak to rozwiązuje nie znaczy że to jest ok :) trzymaj się zasady KISS. Visual studio robi sobie małe obejście na "wszelki wypadek". Ty tego nie musisz robić